.github/workflows/frontend-publish.yml

```diff
@@ -0,0 +1,75 @@
+name: Build Frontend
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  push:
+    tags: [ 'v*.*.*' ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  # Use docker.io for Docker Hub if empty
+  REGISTRY: ghcr.io
+  # github.repository as <account>/<repo>
+  #IMAGE_NAME: ${{ github.repository }}
+  IMAGE_NAME: frontend
+  REPO: ${{ github.repository }}
```

Comment:
@kalenarndt I don't think this will work as I believe the Help Desk has to create the first containers and only then can we push to update the container versions. Unless things have changed, but that was the experience in another internal repo. But I could be wrong 😄

Reply:
I have no idea. Do you know who to ask for this @karl-cardenas-coding ?